OpenQA/Benchmark/Stopwatch.pm

```diff
@@ -2,6 +2,8 @@ package OpenQA::Benchmark::Stopwatch;
 
 use strict;
 use warnings;
+use feature 'signatures';
+no warnings 'experimental::signatures';
```

Comment:
You forgot this one? :)

Reply:
no, my sed expressions actually do not use the Mojo way so the PR draft is a mix of both to show the alternatives to reviewers. I will go with Mojo::Base now and cover all cases